PhoneBook/Brokers/LoggingBroker.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace PhoneBook.Brokers
+{
+    internal class LoggingBroker : ILoggingBroker
+    {
+        public void LogInformation(string message) =>
+            Console.WriteLine(message); 
+            
```

Comment:
This is odd

Reply:
# Muammo hal qilindi.

![image](https://github.com/AlijonovAzamjon2303/PhoneBook/assets/112892881/1ee252c0-44d3-4674-9e51-87f059fd649b)